integration-tests/src/server.js

```diff
@@ -20,26 +21,44 @@ const app = express()
 const server = http.createServer(app)
 server.setTimeout(serverTimeout)
 
+const firestore = new Firestore()
+
 const testTimeout = async timeout => {
   await delay(timeout)
   throw new Error(`Test timeout after ${timeout}ms`)
 }
 
+
+
 const runTest = fn => withPuppeteer(async ({ browser, context, name, page, req, timeout }) => {
   const targetEnvParams = { ...envs[req.query.targetEnv] }
+  const testGroup = req.query.testGroup
+  const startTime = new Date()
+  const host = req.headers.host
+  const reportTest = async error => {
+    const endTime = new Date()
+    const testReport = { testGroup, runtimeInMilliseconds: endTime - startTime, status: error ? 'error' : 'success', testName: name, error, host, startTime }
```

Comment:
Should we record `endTime` too? I know the 3rd number can be calculated from the other two but it might be useful to record the raw data.

Reply:
Sure! I can add endTime. TOL about use cases, I wanted to have a startTime so we could compare when runs started with other runs. I don't know how much value endTime will add, but its cheap and easy enough to add.